v3/src/models/shared/shared-case-metadata.ts

```diff
@@ -26,7 +27,9 @@ export const SharedCaseMetadata = SharedModel
     hidden: types.map(types.boolean),
     caseTableTileId: types.maybe(types.string),
     caseCardTileId: types.maybe(types.string),
-    lastShownTableOrCardTileId: types.maybe(types.string) // used to restore the last shown tile both have been hidden
+    lastShownTableOrCardTileId: types.maybe(types.string), // used to restore the last shown tile both have been hidden
+    lowAttributeColor: types.optional(types.string, kDefaultLowAttributeColor),
+    highAttributeColor: types.optional(types.string, kDefaultHighAttributeColor)
```

Comment:
This seems to imply that there's a single low-color and single high-color per `SharedCaseMetadata` instance. It seems to me the colors need to be associated with an attribute since multiple graphs can have different colors associated with different attributes on the same `SharedCaseMetadata`. I was expecting something more like:
```typescript
const ColorRangeModel = types.model("ColorRangeModel", {
  lowColor: kDefaultLowAttributeColor,
  highColor: kDefaultHighAttributeColor
})
```
and then in `SharedCaseMetadata`:
```typescript
    // id is attribute id
    numericColors: types.map(ColorRangeModel)
```